src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -824,8 +841,8 @@
             $scope.waitingForImportResponse = true; // Small spinner on
             groupingsService.invalidUhIdentifiers(uhIdentifiers, (res) => { // Check for invalid uhIdentifiers
                 $scope.waitingForImportResponse = false; // Small spinner off
-                $scope.isBatchImport = uhIdentifiers.length > Threshold.MULTI_ADD;
-
+                $scope.isBatchImport = angular.element(document.querySelector("#upload"));
+                $scope.isLargeBatchImport = uhIdentifiers.length > Threshold.MULTI_ADD;
```

Comment:
Is there a way to avoid having to separate a batch import and a large batch import? Preferably, we should only have $scope.isBatchImport. Also, I'm not too keen of using document.querySelector to check if it is a batch import. We should always try to avoid using the DOM functions in Angular.js.

Reply:
I think we need isLargeBatchImport cause it's used to trigger the warning model that you are trying to add more than 100 members either from the text box or a file. Maybe we could rename it to something else because batch import implies it came from an imported file. 
isBatchImport it is set in reader.onload and I had to add some clean up to set it to false depending on how addMembers() went. 

Also, I updated the large imports modal to look like the other add modals with the warning about how it takes 3-5 minutes for it to update.